pip_audit/service/service.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC
+from dataclasses import dataclass
+from typing import Dict, List, Optional
+
+from packaging.version import Version
+
+
+@dataclass(frozen=True)
```

Comment:
I figured these would be handy since they generate some convenience methods and they're a form of documentation to indicate:
- That it's a POD type
- That it's immutable

It's not included in earlier Python versions though so I had to add it as a dependency. I can easily remove it if you'd like.

Reply:
LGTM! I was also going to use dataclasses, for the same reasons 🙂 . Python 3.6 will be EOLed soon anyways, at which point we can remove the dep.